scheduler/test/cook/test/scheduler/scheduler.clj

```diff
@@ -1646,122 +1735,83 @@
           (is (= :end-marker (async/<!! offers-chan)))
           (is (= 1 (count @launched-offer-ids-atom)))
-          (is (= 1 (count @launched-job-ids-atom)))
-          (is (= #{"job-1"} (set @launched-job-ids-atom)))))
+          (is (= 1 (count @launched-job-names-atom)))
+          (is (= #{"job-1"} (set @launched-job-names-atom)))))
 
-      (testing "offer for first three normal jobs"
+      (testing "offer for first three jobs"
         (let [num-considerable 10
               offers [offer-3]]
-          (is (run-handle-resource-offers! num-considerable offers :normal))
+          (is (run-handle-resource-offers! num-considerable offers "test-pool"))
           (is (= :end-marker (async/<!! offers-chan)))
           (is (= 1 (count @launched-offer-ids-atom)))
-          (is (= 3 (count @launched-job-ids-atom)))
-          (is (= #{"job-1" "job-2" "job-3"} (set @launched-job-ids-atom)))))
+          (is (= 3 (count @launched-job-names-atom)))
+          (is (= #{"job-1" "job-2" "job-3"} (set @launched-job-names-atom)))))
 
-      (testing "offer not fit for any normal job"
+      (testing "offer not fit for any job"
         (let [num-considerable 10
               offers [offer-5]]
-          (is (run-handle-resource-offers! num-considerable offers :normal))
-          (is (= :end-marker (async/<!! offers-chan)))
+          (is (run-handle-resource-offers! num-considerable offers "test-pool"))
           (is (zero? (count @launched-offer-ids-atom)))
-          (is (empty? @launched-job-ids-atom))))
+          (is (= :end-marker (async/<!! offers-chan)))
+          (is (empty? @launched-job-names-atom))))
 
       (testing "offer fit but user has too little quota"
         (let [num-considerable 10
               offers [offer-1 offer-2 offer-3]
               user-quota {test-user {:count 5, :cpus 4, :mem 4096, :gpus 0}}]
-          (is (run-handle-resource-offers! num-considerable offers :normal :user-quota user-quota))
+          (is (run-handle-resource-offers! num-considerable offers "test-pool" :user-quota user-quota))
           (is (= :end-marker (async/<!! offers-chan)))
           (is (zero? (count @launched-offer-ids-atom)))
-          (is (empty? @launched-job-ids-atom))))
+          (is (empty? @launched-job-names-atom))))
 
       (testing "offer fit but user has capacity usage"
         (let [num-considerable 10
               offers [offer-1 offer-2 offer-3]
               user->usage {test-user {:count 10, :cpus 50, :mem 32768, :gpus 10}}]
-          (is (run-handle-resource-offers! num-considerable offers :normal :user->usage user->usage))
+          (is (run-handle-resource-offers! num-considerable offers "test-pool" :user->usage user->usage))
           (is (= :end-marker (async/<!! offers-chan)))
           (is (zero? (count @launched-offer-ids-atom)))
-          (is (empty? @launched-job-ids-atom))))
-
-      (testing "gpu offers for all gpu jobs"
```

Comment:
Should this test and the next test be copied under the mesos deftest? I see them under k8s.

Reply:
it's redundant to include it for mesos because mesos does not have gpu offers and does not allow jobs to request gpus. so there is a case under the mesos deftest where all offers are being considered for all jobs and the assertion is that only non-gpu jobs get matched.

Reply:
Thanks for discussion. I'm good.